test/go.test.ts

```diff
@@ -804,7 +801,7 @@ It returns the number of bytes written and any write error encountered.
 			return vscode.window.showTextDocument(textDocument).then(editor => {
 				let promises = testCases.map(([position, expected]) =>
 					provider.provideCompletionItems(editor.document, position, null).then(items => {
-						let labels = items.map(x => x.label);
+						let labels = items.items.map(x => x.label);
```

Comment:
I merged another PR that fixed the tests, can you revert your changes to this file?